CHANGELOG.md

```diff
@@ -1,9 +1,27 @@
 ## master
 
-* Relay will now throw an invariant if range behavior keys are unsorted.
+## 0.6.0 (December 4, 2015)
+
+* Bump the `babel-relay-plugin` version to v0.5.0 (now Babel 6 compatible).
```

Comment:
should we bump to 0.6.0 - it will make it *much* less confusing to users if the Relay & plugin versions match.